packages/web-runtime/src/components/VersionCheck.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div v-if="isLoading" class="version-check-loading flex items-center">
+    <span v-text="$gettext('Checking for updates')" />
+    <oc-spinner class="ml-1" size="xsmall" />
+  </div>
+  <div v-else>
+    <div v-if="!updateAvailable" class="version-check-no-updates flex items-center">
```

Comment:
We also need an error case here. If the request fails (e.g. because of a missing `connect-src` entry in the CSP rules), we can't show `Up to date`. We should hint towards an error requesting the most recent version. `Version check failed` or something similar (short!) would be fine.